app/services/food_service.rb

```diff
@@ -0,0 +1,34 @@
+class FoodService
+  def self.search_foods(search)
+    conn = Faraday.new(url: 'https://api.nal.usda.gov') do |faraday|
+      # faraday.headers['api_key'] = ENV['movie_api_key']
+    end
+    query = search
+    # query[' '] = '%20'
+    query.gsub(' ', "%20")
```

Comment:
Nice solution to removing spaces, but Faraday should actually be smart enough to handle the spaces without breaking